flow/scripts/cts.tcl

```diff
@@ -79,14 +79,14 @@ append_env_var additional_args SKIP_PIN_SWAP -skip_pin_swap 0
 append_env_var additional_args SKIP_GATE_CLONING -skip_gate_cloning 0
 
 if {[info exists ::env(SKIP_CTS_REPAIR_TIMING)] == 0 || $::env(SKIP_CTS_REPAIR_TIMING) == 0} {
-  if { [info exists ::env(EQUIVALENCE_CHECK)] } {
+  if {[info exists ::env(EQUIVALENCE_CHECK)] && $::env(EQUIVALENCE_CHECK) == 1} {
       write_eqy_verilog 4_before_rsz.v
   }
 
   puts "repair_timing [join $additional_args " "]"
   repair_timing {*}$additional_args
 
-  if { [info exists ::env(EQUIVALENCE_CHECK)] } {
```

Comment:
Instead of modifying the `cts.tcl` which applies to all designs including the ones outside of AT, you could modify the `distributed.py` by adding `EQUIVALENCE_CHECK=0` to the `make_command` in the `openroad` function, for example.

Reply:
So `EQUIVALENCE_CHECK` is meant more as a flag rather than a switch?

Reply:
If you are using a definition where "flag" is true/false and "switch" can have multiple options, then yes it is a flag. If that is not what you meant, please let me know.

Reply:
I am just clarifying on the use of `EQUIVALENCE_CHECK` because I see in `config.mk` files they are defined as `export ... ?= 1`, but in the code `cts.tcl` we just check for the existence of such a variable, so it seems that the code treats it more like a flag.

Reply:
Code checks for existence of `EQUIVALENCE_CHECK`, so even if it is set to zero I think the logic should still execute?

Reply:
> I am just clarifying on the use of `EQUIVALENCE_CHECK` because I see in `config.mk` files they are defined as `export ... ?= 1`, but in the code `cts.tcl` we just check for the existence of such a variable, so it seems that the code treats it more like a flag.

Apologies, I think I confused the diffs. I think is ok to add the check `== 1`.



> As it is off by default I don't think we need to worry about explicitly turning it off.

Jack is facing an issue with one design that has `eqy` check enabled, but is failing a lot during AT. I suggested to disable `eqy` for AT runs while we investigate the issue.